api/lib/domain/models/User.js

```diff
@@ -37,6 +40,22 @@ class User {
     hasBeenAnonymised,
     hasBeenAnonymisedBy,
   } = {}) {
+    if (locale) {
+      try {
+        locale = Intl.getCanonicalLocales(locale)[0];
+      } catch (error) {
+        throw new LocaleFormatError(locale);
+      }
+
+      if (locale === 'en-GB') {
+        locale = 'en';
+      }
+
+      if (!SUPPORTED_LOCALES.includes(locale)) {
+        throw new LocaleNotSupportedError({ locale, supportedLocales: SUPPORTED_LOCALES });
```

Comment:
Suite à notre dernière mob ensemble on n'utilise plus d'autres propriétés que `locale` : 

```suggestion
        throw new LocaleNotSupportedError(locale);
```